framework/src/mesh/MooseMesh.C

```diff
@@ -1228,18 +1212,18 @@ MooseMesh::detectPairedSidesets()
   // Container to catch IDs passed back from the BoundaryInfo object
   std::vector<BoundaryID> bounds_ids;
 
-  for (unsigned int curr_dim=0; curr_dim < dim; ++curr_dim)
+  for (unsigned int curr_dim = 0; curr_dim < dim; ++curr_dim)
   {
     std::pair<BoundaryID, BoundaryID> paired_boundary;
-    for (unsigned int i=0; i<2; ++i)
+    for (unsigned int i = 0; i<2; ++i)
     {
       std::map<BoundaryID, unsigned int> boundary_counts;
-      for (unsigned int j=0; j<2; ++j)
-        for (unsigned int k=0; k<z_dim; ++k)
+      for (unsigned int j = 0; j<2; ++j)
+        for (unsigned int k = 0; k<z_dim; ++k)
         {
           boundary_info.boundary_ids(_extreme_nodes[i*strides[curr_dim][0] + j*strides[curr_dim][1] + k*strides[curr_dim][2]], bounds_ids);
 
-          for (unsigned int l=0; l<bounds_ids.size(); ++l)
+          for (unsigned int l = 0; l<bounds_ids.size(); ++l)
```

Comment:
I missed a bunch. Let me fix all the ones I see and repush before you waste
time looking. I have a better method now.

On Thu, Mar 24, 2016 at 3:16 PM Daniel Schwen <notifications@github.com>
wrote:

> In framework/src/mesh/MooseMesh.C
> <https://github.com/idaholab/moose/pull/6639#discussion_r57389065>:
>
> >          {
> >            boundary_info.boundary_ids(_extreme_nodes[i*strides[curr_dim][0] + j*strides[curr_dim][1] + k*strides[curr_dim][2]], bounds_ids);
> >
> > -          for (unsigned int l=0; l<bounds_ids.size(); ++l)
> > +          for (unsigned int l = 0; l<bounds_ids.size(); ++l)
>
> here and two above as well
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/6639/files/0704ae5ce8c7ba7fcbc1cc65ee33c384705dccdc#r57389065>
>